File: python/1048.py
salary = float(input())

percentage = 0

if salary <= 400:
    percentage = 15
elif salary > 400 and salary <= 800:
    percentage = 12
elif salary > 800 and salary <= 1200:
    percentage = 10
elif salary > 1200 and salary <= 2000:
    percentage = 7 
elif salary > 2000:
    percentage = 4 
    
addition = (salary * percentage / 100)
newSalary = salary + addition
print(f'Novo salario: {newSalary:.2f}')
print(f'Reajuste ganho: {addition:.2f}')
print(f'Em percentual: {percentage} %')

File: lua/1048.lua
local originalSalary = io.read('n')
local newSalary
local percent
local addition

if originalSalary <= 400 then
    percent = 15
elseif originalSalary <= 800 then
    percent = 12
elseif originalSalary <= 1200 then
    percent = 10
elseif originalSalary <= 2000 then
    percent = 7
else
    percent = 4
end

addition = originalSalary / 100 * percent
newSalary = originalSalary + addition

print('Novo salario: '..string.format('%.2f', newSalary))
print('Reajuste ganho: '..string.format('%.2f', addition))
print('Em percentual: '..percent..' %')